crystalBox.py

```diff
@@ -307,6 +309,37 @@ def getEquivalents(self, hkl):
         print(sg)
         print(pg)
         print("The equivalents of reflection", hkl, "include the following:", equivalents)    
+    
+    def makeCrystal(self):
+        params = [self.a_orig,
+                  self.b_orig,
+                  self.c_orig,
+                  self.alpha_orig,
+                  self.beta_orig,
+                  self.gamma_orig]
+        #print(params)
+        line = ' '.join(['{}']*6)
+        constants = line.format(*params)
+        scatterers = self.get_scatterers()
+        atom_info = ';'.join([line.format(*s) for s in scatterers])
+        crystal_structure = CrystalStructure(constants,self.HMSymbol, atom_info)
+        
+        return crystal_structure
+                        
+    def cartesianHKL(self,h):
```

Comment:
I feel this would benefit from a validation step that makes sure h is the correct dimenions i.e. (1x3) or (3x1) whichever matmul needs to work.